### ISSUE TITLE
Support slack/mattermost webhook 

### ISSUE THREAD

**Issue by faroit** [audiolabs/conferences#5]
every time a new Event is created a msg should be posted to the chat system.

**Comment by nils-werner** [comment]
Also (if not more) interesting: A message when people

 - Plan to go
 - Plan to submit
 - Were accepted

**Comment by faroit** [comment]
I agree. But I would like to implement #4 first so that it becomes easier to use to submit your own status
